README.md

```diff
@@ -8,36 +9,65 @@ Part of this documentation (the `Reference` section) is generated from `json` ob
 Every commit and pull request to this repository will trigger a new build of the documentation. The `master` will build into the "latest" version and each versioned tag is selectable in the flyout menu, with the last version being marked as "stable".
 
 ## Building the documentation locally
+
 During the development it is beneficial to build the documentation page locally to see the results before committing something into the repository that fails to build or that produces warnings.
 
-  1. In a shell, move to this directory.
-  2. Run `python -m venv .venv` to create a python virtual environment.
-  3. Run `.\.venv\Scripts\activate` to activate the python virtual environment.
-  4. `pip install -r requirements.txt` (this is technically only necessary for the first time or when the dependencies change, but it is a good habit to run this every time after a checkout).
-  5. Run `make html`.
+1. In a shell, move to this directory.
+2. Run `python -m venv .venv` to create a python virtual environment.
+3. Run `.\.venv\Scripts\activate` to activate the python virtual environment.
+4. `pip install -r requirements.txt` (this is technically only necessary for the first time or when the dependencies change, but it is a good habit to run this every time after a checkout).
+5. (optional) Generate the files for the OpenSpace reference, and alternatively also the asset examples if you want to use versions different than the ones on the origin/master branch. See section "Generate OpenSpace reference" below for details.
+6. Run `make html`/`.\make.bat html`.
 
 This will create a `build/html` folder structure that contains a `index.html` which you can open in a webbrowser to view the documentation.
 
 Before committing to the repository it can also be beneficial to run `./make.bat linkcheck` which checks whether all links referenced in the document are actually correct or if we have links that are now dead.
 
-## Make commands
+### Make commands
 
 - `make html` will build the documentation locally.
 - `make clean` will clean out the build folder and remove everything built previously.
 - `make linkcheck` will check so that all links are correct.
 
+### Generate OpenSpace reference (optional)
+
+If you have updated documentation for the Scripting API or the Asset Components (including example files) in the engine, you might want to view these changes in the documentation. The following steps describe how to do this.
+
+1. Set the `genereate_reference` variable in `conf.py` to `True`. This will result in the files being regenerated when you run `make html`. Otherwise, the generation is only done if the files do not already exist.
```

Comment:
```suggestion
1. Set the `generate_reference` variable in `conf.py` to `True`. This will result in the files being regenerated when you run `make html`. Otherwise, the generation is only done if the files do not already exist.
```